config/config.go

```diff
@@ -0,0 +1,117 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+)
+
+const (
+	BaseID   = "base_ID" //required
+	TableID  = "table_ID"
+	ViewID   = "view_ID"
+	RecordID = "record_ID"
+)
+
+var (
+	ErrEmptyConfig = errors.New("missing or empty config")
+)
+
+type Config struct {
+	baseID   string
+	tableID  string
+	viewID   string
+	recordID string
+}
+
+func ParseBaseConfig(cfg map[string]string) (Config, error) {
+
+	err := checkEmpty(cfg)
+	if err != nil {
+		return Config{}, fmt.Errorf("map must not be empty")
+	}
+
+	BaseID, ok := cfg[BaseID]
+	if !ok {
+		return Config{}, fmt.Errorf("%q config value must be set", BaseID)
+	}
+	err = checkFormat(BaseID, "app")
```

Comment:
Is the prefix required by Airtable?

Reply:
Yep! When removing them from the URL the page can not be found.